<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>86</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_bank">
       <property name="text">
        <string>Select Region of Interest:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="combo_bank">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <item>
        <property name="text">
         <string>Custom Grouping File</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1 (North)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2 (South)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Crop to Spectra</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Texture (20 spec)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Texture (30 spec)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QWidget" name="widget_crop" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_crop">
          <property name="text">
           <string>Custom Spectra:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edit_crop">
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_cropValid">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>10</horstretch>
            <verstretch>20</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>10</width>
            <height>0</height>
           </size>
          </property>
          <property name="palette">
           <palette>
            <active>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>170</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>170</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>118</red>
                <green>116</green>
                <blue>108</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QWidget" name="widget_custom" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="FileFinderWidget" name="finder_custom" native="true">
          <property name="minimumSize">
           <size>
            <width>496</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileFinderWidget</class>
   <extends>QWidget</extends>
   <header>mantidqt.widgets.filefinderwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>combo_bank</tabstop>
  <tabstop>edit_crop</tabstop>
  <tabstop>finder_custom</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
